docs/releases/202404.mdx

```diff
@@ -4,6 +4,13 @@ title: April 2024
 displayed_sidebar: releaseSideBar
 ---
 
+### April 25 2024
+**Features/Updates**
+- The advanced search panel on the file browser now includes file metadata filters, so that a file search can be constructed in a single pane.
+
```

Comment:
should probably add in the new nodes that are there - there are 5 - WSP, LC, LC Multi, MS, MS Multi

Reply:
added those in